packages/types/src/clerk.ts

```diff
@@ -894,6 +894,11 @@ export type UserButtonProps = UserButtonProfileMode & {
    * Controls the default state of the UserButton
    */
   defaultOpen?: boolean;
+  /**
+   * By default, the popover will be rendered outside of the app root and into the body.
+   * @default true
```

Comment:
I'm not seeing where this is being defaulted to `true`. Is that somewhere outside this PR? I tried to track it down, but our Context setup seems a bit complex 😵‍💫

Reply:
I had the same question, the existing code defines it so it's hard to see it in the PR. It is defined [here](https://github.com/clerk/javascript/blob/a6620ae2e8a871639ee8b00e3677b63b179663a8/packages/clerk-js/src/ui/elements/Popover.tsx#L16)